src/app/app.module.ts

```diff
@@ -3,11 +3,15 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { ProductsComponent } from './modules/products/products.component';
+import { ProductsModule } from './modules/products/products.module';
+import { ProductsService } from './modules/products/products.service';
+import { ProductComponent } from './modules/product/product.component';
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule],
-  providers: [],
+  declarations: [AppComponent, ProductsComponent, ProductComponent],
+  imports: [BrowserModule, AppRoutingModule, ProductsModule],
+  providers: [ProductsService],
```

Comment:
You can replace this, on the setting of the service, describe how to do in the service, it is possible to do differently, not a mistake.